assets/activation-scripts/activate

```diff
@@ -177,6 +194,12 @@ else
 
 fi
 
+# Start services before the shell or command is invoked
+if [ "$FLOX_ACTIVATE_START_SERVICES" == "true" ]; then
+  config_file="$FLOX_ENV/service-config.yaml"
+  "$_process_compose" up -f "$config_file" -u "$_FLOX_SERVICES_SOCKET" --tui=false >/dev/null 2>&1 &
```

Comment:
(nb) For the benefit of other reviewers, some features are covered by separate tickets:
- https://github.com/flox/flox/issues/1640
- https://github.com/flox/flox/issues/1584